libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -162,7 +162,8 @@ export class LastInstallFlagFactory {
    */
   public static getCommonTempFlag(
     rushConfiguration: RushConfiguration,
-    extraState: Record<string, string> = {}
+    extraState: Record<string, string> = {},
+    subspaceName: string | undefined
```

Comment:
A non-optional parameter cannot follow an optional parameter.

Reply:
Changed the order